master/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy.md

```diff
@@ -152,10 +152,10 @@ documentation.
 ## Multiple networks
 
 While some network providers tend to use multiple networks to enforce
-isolation, Calico philosophy instead opts to put all containers in the same,
+isolation, {{site.prodname}} instead opts to put all containers in the same,
```

Comment:
missing philosophy?

Reply:
I purposefully removed "philosophy". I think it is subtly incorrect here because a "philosophy" cannot actually opt to do anything. But also, I think this word is gratuitous/unnecessary in this sentence. Thanks though @doublek appreciate you taking a look!